src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <!-- FONTS -->
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <article class="product" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="IMac image" 
+        class="product__img"
+      >
+
+      <h3 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+      <p class="product__code">
+        Product code:
+        <span id="product-code">195434</span>
```

Comment:
```suggestion
        <span id="product-code">195434</span>
```

why do you need id?

Reply:
I thought that this is an information that loads from database usually, so I created a container to refer to in scripts